js/popup.js

```diff
@@ -1,486 +1,462 @@
-		$("#edt_record_a_measurement_block").show();
-		$("#edt-addmeasurement-variable-name").val(name);
+    }
 
-	}
+    var n_value = getVariableWithName(name);
 
-}
+    if (n_value == null) {
+        $("#record_a_measurement_block").hide();
+        $("#add_record_a_measurement_block").show();
+        $("#add-addmeasurement-variable-name").val(name);
 
+    }
+    else {
+        $("#record_a_measurement_block").hide();
+        $("#edt_record_a_measurement_block").show();
+        $("#edt-addmeasurement-variable-name").val(name);
+
+    }
+
+}
 
 
 // registration button clicked
-var btnQuantiModoRegisterClick = function()
-{
-	var optionsUrl = chrome.extension.getURL("options/options.html"); 
-	//var content = '<a href="' + optionsUrl + '" target="_blank">Options</a>';
+var btnQuantiModoRegisterClick = function () {
+    var optionsUrl = chrome.extension.getURL("options/options.html");
+    //var content = '<a href="' + optionsUrl + '" target="_blank">Options</a>';
 
-	chrome.tabs.create({ url: optionsUrl });
+    chrome.tabs.create({url: optionsUrl});
 };
 
 
 // registration button clicked
-var btnQuantiModoLogoCorrelateClick = function()
-{
-	chrome.tabs.create({ url: "https://app.quantimo.do/correlate/" });
+var btnQuantiModoLogoCorrelateClick = function () {
+    chrome.tabs.create({url: "https://app.quantimo.do/correlate/"});
 };
 
 
 // registration button clicked
-var onRegisterAClicked = function()
-{
-	chrome.tabs.create({ url: "https://app.quantimo.do/citizen-science/register/" });
+var onRegisterAClicked = function () {
+    chrome.tabs.create({url: "https://app.quantimo.do/citizen-science/register/"});
 };
 
 // facebook button clicked
-var onQmFacebookButtonClicked = function()
-{
-	chrome.tabs.create({ url: "https://app.quantimo.do/wp-login.php?action=wordpress_social_authenticate&provider=Facebook&redirect_to=https%3A%2F%2Fquantimo.do%2Fwp-login.php%3Fredirect_to%3Dhttps%253A%252F%252Fquantimo.do" });
+var onQmFacebookButtonClicked = function () {
+    chrome.tabs.create({url: "https://app.quantimo.do/wp-login.php?action=wordpress_social_authenticate&provider=Facebook&redirect_to=https%3A%2F%2Fquantimo.do%2Fwp-login.php%3Fredirect_to%3Dhttps%253A%252F%252Fquantimo.do"});
 };
 
 // google button clicked
-var onQmGoogleButtonClicked = function()
-{
-	chrome.tabs.create({ url: "https://app.quantimo.do/wp-login.php?action=wordpress_social_authenticate&provider=Google&redirect_to=https%3A%2F%2Fquantimo.do%2Fwp-login.php%3Fredirect_to%3Dhttps%253A%252F%252Fquantimo.do" });
+var onQmGoogleButtonClicked = function () {
+    chrome.tabs.create({url: "https://app.quantimo.do/wp-login.php?action=wordpress_social_authenticate&provider=Google&redirect_to=https%3A%2F%2Fquantimo.do%2Fwp-login.php%3Fredirect_to%3Dhttps%253A%252F%252Fquantimo.do"});
 };
 
 // Simple Sign In button clicked
-var onQmSignButtonClicked = function()
-{
-	chrome.tabs.create({ url: "https://app.quantimo.do/analyze" });
+var onQmSignButtonClicked = function () {

```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['onQmSignButtonClicked' was used before it was defined.](https://www.codacy.com/app/m_3/QuantiModo-Chrome-Extension/file/832637814/issues/source?bid=2779649&fileBranchId=2790922#l140)